twingraph/src/main/kotlin/com/cosmotech/twingraph/api/TwingraphServiceImpl.kt

```diff
@@ -164,6 +164,35 @@ class TwingraphServiceImpl(
     return resultSet.toJsonString()
   }
 
+  override fun updateGraphMetaData(
+      organizationId: String,
+      graphId: String,
+      requestBody: Map<String, String>
+  ): Any {
+    val organization = organizationService.findOrganizationById(organizationId)
+    csmRbac.verify(organization.getRbac(), PERMISSION_READ)
+
+    if (requestBody.containsKey("graphRotation")) {
```

Comment:
You don't need to check if `requestBody` contains `graphRotation` key as get operator return `null` if not found (see. https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-map/get.html)
You can rewrite code like below:
```
      val graphRotation = requestBody["graphRotation"]?.toInt()
      if (graphRotation == null || graphRotation < 1) {
        throw CsmClientException("GraphRotation should be a positive integer")
      }
```